openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/CustomEdge.component.tsx

```diff
@@ -30,7 +34,7 @@ export const CustomEdge = ({
   data,
   selected,
 }: EdgeProps) => {
-  const { onEdgeClick, ...rest } = data;
+  const { onEdgeClick, addPipelineClick, ...rest } = data;
```

Comment:
```suggestion
  const { onEdgeClick, addPipeline, ...rest } = data;
```

Reply:
I think this should be fine as I am indicating click for add pipeline button

Reply:
That is the callback name, so I think we don't need to name it with the `Click` suffix.

Reply:
Got it.